src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_sun calendar--month-length_31">
```

Comment:
The `calendar--start-day_sun` and `calendar--month-length_31` classes seem to be modifiers in BEM methodology. If they are meant to be modifiers, they should be used along with the block or element they are modifying. Make sure you are using BEM methodology correctly.